common/command_line_test.cpp

```diff
@@ -188,6 +188,9 @@ TEST(ArgParserTest, ShortArgs) {
   EXPECT_THAT(integer_option, Eq(123));
 
   TestRawOstream os;
+
+  EXPECT_THAT(parse({"-v"}, os), IsError(StrEq("unknown short option `-v`")));
```

Comment:
```suggestion
  EXPECT_THAT(parse({"-v"}, os), IsError(StrEq("unknown short option `-v`")));
  EXPECT_THAT(parse({"-xvx"}, os), IsError(StrEq("unknown short option `-v`")));
```
Might be nice to test that we diagnose only the one bad short option, not the whole thing.